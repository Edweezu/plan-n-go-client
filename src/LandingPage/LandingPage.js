import React from 'react'
import { Link } from 'react-router-dom'

class LandingPage extends React.Component {
    render () {
        return (
            <main className='landing-page container'>
                <header>
                    <h1>
                        Plan n Go
                    </h1>
                </header>
                <section className='landing-descript-main'>
                    <h2>Trip Planning Simplified</h2>
                    <p>
                    Many times, planning trips, especially long trips, is often times the most stressful and arduous part of the journey. Plan n Go helps simplify the experience by keeping all your trip details in one spot. You can also edit flight paths and add items onto your endless packing list all on the app. Get started now by simply creating an account! 
                    </p>
                    <Link className="link btn" to='/register'>Create an Account</Link>
                </section>
                <section className='landing-descript-two'>
                    <h2>Create Lavish trip Plans</h2>
                    <p>
                    Plan and go on that dream vacation you've always wanted. Get up to date weather details, set crucial flight times, and create detailed daily itineraries for your vacation. 
                    </p>
                </section>
                <section className='landing-descript-two'>
                    <h2>Your Personal Packing List</h2>
                    <p>
                    Have you ever realized that you've forgotten something a couple hours after leaving the house for a trip? Never forget again with our very intuitive packing list.
                    </p>
                </section>
            </main>
        )
    }
}

export default LandingPage;