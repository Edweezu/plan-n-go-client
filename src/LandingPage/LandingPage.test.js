import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import LandingPage from './LandingPage'


describe('LandingPage component', () => {
    it('renders the given component', () => {
        const wrapper = shallow(<LandingPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})