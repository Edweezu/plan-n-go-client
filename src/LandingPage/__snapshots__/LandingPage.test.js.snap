// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the given component 1`] = `
<main
  className="landing-page container"
>
  <header
    className="landing-page-header"
  >
    <h1>
      Trip Planning Simplified
    </h1>
    <Link
      className="link btn"
      to="/register"
    >
      Create an Account
    </Link>
  </header>
  <h2
    className="landing-descript-header"
  >
    Plan-n-Go Functionalities
  </h2>
  <section
    className="landing-descript-container"
  >
    <section
      className="landing-descript-one"
    >
      <div
        className="landing-descript-img-data"
      >
        <img
          alt="secure data"
          className="descript-img"
          src="secure-data.jpg"
        />
      </div>
      <p
        className="landing-description"
      >
        Manage Countless Amount of Trips in One Place while knowing that your Data is Secure.
      </p>
    </section>
    <section
      className="landing-descript-two"
    >
      <div
        className="landing-descript-img-data"
      >
        <img
          alt="secure data"
          className="descript-img"
          src="itinerary.jpg"
        />
      </div>
      <p
        className="landing-description"
      >
        Create Lavish Trip Excursions with our Detailed Travel Itinerary.
      </p>
    </section>
    <section
      className="landing-descript-two"
    >
      <div
        className="landing-descript-img-data"
      >
        <img
          alt="secure data"
          className="descript-img"
          src="packing-list.jpg"
        />
      </div>
      <p
        className="landing-description"
      >
        Check, Add, and Delete Items off our Intuitive Packing List.
      </p>
    </section>
  </section>
  <section
    className="landing-page-reminder"
  >
    <Link
      className="link btn"
      to="/register"
    >
      Get Started
    </Link>
  </section>
</main>
`;
